resources/snap_packs/responses/snap_pack.yml

```diff
@@ -0,0 +1,68 @@
+application/json:
+  schema:
+    $ref: "../models/snap_pack.yml"
+  example:
+    id: ord_0d6a16a3fff6318ac8f8008dc1
+    description: April Campaign
+    to:
+      id: adr_bae820679f3f536b
+      description:
+      name: HARRY ZHANG
+      company:
+      phone:
+      email:
+      address_line1: 210 KING ST STE 6100
+      address_line2:
+      address_city: SAN FRANCISCO
+      address_state: CA
+      address_zip: 94107-1741
+      address_country: UNITED STATES
+      metadata: {}
+      date_created: "2017-09-05T17:47:53.767Z"
+      date_modified: "2017-09-05T17:47:53.767Z"
+      deleted: true
+      object: address
+    from:
+      id: adr_210a8d4b0b76d77b
+      description:
+      name: LEORE AVIDAR
+      company:
+      phone:
+      email:
+      address_line1: 210 KING ST STE 6100
+      address_line2:
+      address_city: SAN FRANCISCO
+      address_state: CA
+      address_zip: 94107-1741
+      address_country: UNITED STATES
+      metadata: {}
+      date_created: "2017-09-05T17:47:53.767Z"
+      date_modified: "2017-09-05T17:47:53.767Z"
+      deleted: true
+      object: address
+    url: https://lob-assets.com/order-creatives/ord_0d6a16a3fff6318ac8f8008dc1_comp_a20fd48ba4efda76ee827400d.pdf?version=v1&expires=1618512040&signature=qvyCqXI1ndBvc4AjvG8FlirqLXEcfmYo4sDrRtabaXMOsX88to9G3K49uIk_aqevvZXe8HoRYD_nWydbQHqaCA
```

Comment:
The url is constructed slightly differently to lob-api - `psc_blah` vs `order-creatives/ord_123_comp_456`